source

```diff
@@ -96615,18 +96627,22 @@ new PaymentRequest(&hellip;); // Allowed to use
    <li><p><span data-x="abort a document">Abort</span> the <span data-x="nav-document">active
    documents</span> of each of <var>document</var>'s <span>descendant navigables</span>. <span
    class="XXX" data-x="">In what order?</span> If this results in any of those <code>Document</code>
-   objects having their <i data-x="concept-document-salvageable">salvageable</i> state set to false,
-   then set <var>document</var>'s <i data-x="concept-document-salvageable">salvageable</i> state to
-   false also.</p></li>
+   objects having their <i data-x="concept-document-salvageable">salvageable</i> state set to false
```

Comment:
I don't think we need to make the children unsalvageable from here, since if they make themselves state set to false, then they would have the reasons set already too?